JustSaying.Tools/Commands/MoveCommand.cs

```diff
@@ -62,9 +63,9 @@ public bool Execute()
             return true;
         }
 
-        private void EnsureQueueExists(SqsQueueByName queue)
+        private async Task EnsureQueueExists(SqsQueueByName queue)

```

Comment:
Should this not be called `EnsureQueueExistsAsync`?

Reply:
Thanks @JosephWoodward, this has been addressed